common/djangoapps/config_models/tests/tests.py

```diff
@@ -104,6 +109,53 @@ def test_always_insert(self, __):
         self.assertEquals(2, ExampleConfig.objects.all().count())
 
 
+class ConfigurationModelEqualityTests(TestCase):
+
+    def setUp(self):
+        super(ConfigurationModelEqualityTests, self).setUp()
+        self.user = User()
+        self.user.save()
+
+    def test_equality(self):
+        config = ExampleConfig(changed_by=self.user, string_field='first')
+        config.cache_timeout = 0
```

Comment:
Why are your new methods mocked out when using a mock cache? Assuming you make the mock cache always return `None`, it shouldn't affect your code.